recipes/notices/top_banner_info/top_banner_info_variants.story.vue

```diff
@@ -12,7 +12,11 @@
         <span class="d-fw-bold"> Active Call </span> 3:21
       </template>
       <div class="d-d-flex d-c-pointer">
```

Comment:
Add `d-ai-center` to get `d-d-flex d-c-pointer d-ai-center` (aligning text and icon to middle).